tests/components/eheimdigital/test_light.py

```diff
@@ -127,15 +128,29 @@ async def test_turn_off(
     )
     await hass.async_block_till_done()
 
+    classic_led_ctrl_mock.turn_off.side_effect = EheimDigitalClientError
+
+    with pytest.raises(HomeAssistantError) as exc_info:
+        await hass.services.async_call(
+            LIGHT_DOMAIN,
+            SERVICE_TURN_OFF,
+            {ATTR_ENTITY_ID: "light.mock_classicledcontrol_e_channel_0"},
+            blocking=True,
+        )
+
+    assert isinstance(exc_info.value.__cause__, EheimDigitalClientError)
```

Comment:
I kinda don't care what exception was the root cause, I care about how we handle it, so ideally we either match on the translation
```suggestion
    with pytest.raises(HomeAssistantError, match="exception"):
        await hass.services.async_call(
            LIGHT_DOMAIN,
            SERVICE_TURN_OFF,
            {ATTR_ENTITY_ID: "light.mock_classicledcontrol_e_channel_0"},
            blocking=True,
        )
```
or we should check the translation key in the exc_info